build_tools/ci/cpu_comparison/test_template/matmul_elementwise_MxNxK_bf16_f32.mlir

```diff
@@ -0,0 +1,13 @@
+func.func @matmul_elementwise(%arg0: tensor<${M}x${K}xbf16>, %arg1: tensor<${K}x${N}xbf16>, %arg2: tensor<${N}xf32>) -> tensor<${M}x${N}xf32>
```

Comment:
I make it separate because the first is for int data type, and second is for float, and cannot put them into the same template. The reason is the constant value should be different (0 vs 0.0), and also for `addf vs addi` operations.

Reply:
But the biases are different shapes too? I think "elementwise" is overloaded term

Reply:
I've made more changes to the template files and names. Please review it again. Thanks!

Reply:
Thanks